soweego/linker/evaluate.py

```diff
@@ -116,31 +113,35 @@ def cli(
             predictions_out,
             dir_io,
         )
-        sys.exit(0)
 
-    # Default: average evaluation over k-fold
-    _run_average(
-        classifier,
-        catalog,
-        entity,
-        k_folds,
-        kwargs,
-        performance_out,
-        predictions_out,
-        dir_io,
-    )
+    else:
+        # Default: average evaluation over k-fold
+        _run_average(
+            classifier,
+            catalog,
+            entity,
+            k_folds,
+            kwargs,
+            performance_out,
+            predictions_out,
+            dir_io,
+        )
+
+    sys.exit(0)
```

Comment:
This is probably not needed anymore, since you changed the if statements into if/elif/else

Reply:
you're right, explicit exit removed in 2bb80a9d00839da174f982c93bbf57ab7349b23d